libs/shared-config/src/lib/config.ts

```diff
@@ -366,84 +360,89 @@ export const COURSES = [
   },
   {
     id: 'alumni',
-    label: `I'm a ReDI School alumni (I took a course before)`,
+    label: `I'm a ReDI School alumni (I took a course in the past)`,
     location: 'berlin',
   },
 
   {
     id: 'hamburg_htmlCss',
-    label: `HTML and CSS`,
+    label: `HTML & CSS`,
     location: 'hamburg',
   },
   {
-    id: 'hamburg_introComputerScience',
-    label: `Intro to Computer Science`,
+    id: 'hamburg_javascript',
+    label: `JavaScript`,
     location: 'hamburg',
   },
   {
-    id: 'hamburg_uxUiDesignBasics',
-    label: `UX/UI Design Basics`,
+    id: 'hamburg_introComputerScienceOnline',
+    label: `Introduction to Computer Science (Online)`,
     location: 'hamburg',
   },
-
   {
-    id: 'munich_introComputerScience',
-    label: `Introduction to Computer Science`,
-    location: 'munich',
+    id: 'hamburg_introComputerScienceHybrid',
+    label: `Introduction to Computer Science (Hybrid)`,
+    location: 'hamburg',
   },
   {
-    id: 'munich_pythonIntermediate',
-    label: `Python Intermediate`,
-    location: 'munich',
+    id: 'hamburg_introComputerScienceUkr',
+    label: `Introduction to Computer Science (UKR)`,
+    location: 'hamburg',
   },
   {
-    id: 'munich_frontend1',
-    label: `Frontend 1`,
-    location: 'munich',
+    id: 'hamburg_uxUiDesignBasics',
+    label: `UX/UI Design Basics`,
+    location: 'hamburg',
   },
   {
-    id: 'munich_frontend2',
-    label: `Frontend 2`,
-    location: 'munich',
+    id: 'hamburg_pythonIntermediateOnline',
+    label: `Python Intermediate (Online)`,
+    location: 'hamburg',
   },
   {
-    id: 'munich_frontend_vollzeitkurs',
-    label: 'Frontend Development Vollzeitkurs',
-    location: 'munich',
+    id: 'hamburg_pythonIntermediateHybrid',
+    label: `Python Intermediate (Hybrid)`,
+    location: 'hamburg',
   },
   {
-    id: 'munich_dataAnalytics2',
+    id: 'hamburg_dataAnalytics',
     label: `Data Analytics`,
-    location: 'munich',
+    location: 'hamburg',
   },
   {
-    id: 'munich_backend2',
-    label: `Backend 2`,
-    location: 'munich',
+    id: 'hamburg_backendDevelopment',
+    label: `Back-End Development`,
+    location: 'hamburg',
   },
   {
-    id: 'munich_uxUiDesign',
-    label: `UX/UI Design Basics`,
+    id: 'alumni',
```

Comment:
To uniquely identify this course as the "Hamburg Alumni" course this should be `hamburg_alumni`.

`alumni` by itself means Berlin Alumni.